choicemodels/tools/mergedchoicetable.py

```diff
@@ -93,10 +94,24 @@ class MergedChoiceTable(object):
     random_state : NOT YET IMPLEMENTED
         Representation of random state, for replicability of the sampling.
 
+    sampling_regime : str, [None, 'stratified'], optional
+        Specify the sampling regime for construction of the MergedChoiceTable. Stratified
+        sampling as defined here involves creating a sample by sampling equally from
+        subsamples of the population as identified by membership in a specified group
+        or stratum.
+
+    strata: str, optional
+        If stratified sampling is specified as the sampling regime, then a column name
+        from the alternatives table must be provided upon which stratification will be
+        based. Because equal numbers of samples will be drawn from within each strata,
+        the strata should be distributed roughly evenly across the population, e.g. if
```

Comment:
As long as I have used the same sampling strategy to fit the model, Why does it need to be `distributed roughly evenly across the population`?

Reply:
Only because I wrote the stratification code such that it defines one value for `samp_size_per_strata` (L388) and uses this number to sample from each stratum. A more generalized version of the code would compute strata proportions from the universe of alternatives and then sample from each strata accordingly such that the strata proportions in the sample match those from the universe, but I didn't code that up.